_data/navigation.yml

```diff
@@ -72,16 +72,10 @@
         fr: "/students_fr/"
     - title:
         en: McGill Physics Olympiad Program
-        fr: Programme d'olympiade de physique à McGill
```

Comment:
This file should not have been edited (it is being reverted to a previous version here.)